src/lib/components/Tooltip.svelte

```diff
@@ -74,10 +74,14 @@
     targetIsHovered = false;
   };
 
-  onMount(async () => {
+  const moveTooltipToBody = () => {
     // Move tooltip to the body to avoid it being cut off by overflow: hidden.
     nonNullish(tooltipComponent) && document.body.appendChild(tooltipComponent);
-  });
+  };
+
+  onMount(moveTooltipToBody);
+
+  afterUpdate(moveTooltipToBody);
```

Comment:
Just thinking out loud, would it help to call `moveTooltipToBody` only on hover?

Reply:
If they're not moved, they actually take up space and cause scroll bars where there shouldn't be.
I suppose there may be a way to solve that too. Feel free to give it a try. Let me know if you need help reproducing.